app/ValueObjects/CampMapData.php

```diff
@@ -16,15 +16,18 @@ final class CampMapData
 
     public array $latlng;
 
-    public int $size;
+    public int $num_members;
```

Comment:
Can you pascalCase these. That is more php odomic and in line with the laravel code style.

Reply:
It's actually camelCase, not PascalCase, but sure :)

Reply:
Ahh I always remembered it the other way arround, but thx.